README.md

```diff
@@ -120,7 +120,7 @@ To use as a [ES-module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/
 
 ### More examples
 
-Assuming you have imported minitz as described under 'Installation'.
+The below examples will work only if you have imported minitz as described in the 'Installation' section, if that's not the case results may vary.
```

Comment:
The examples below will work only if you have imported minitz as described in the 'Installation' section.  If that's not the case then results may vary.